app/index.js

```diff
@@ -87,6 +87,16 @@ module.exports = class extends Generator {
 					message: 'A short description of your project:'
 				},
 				{
+					type: 'list',
+					name: 'projectType',
+					message: 'What to you want to build?',
+					choices: [
+						'A static website',
```

Comment:
Better:
`A static website (Static site generator using Handlebars and Frontmatters)`

String must be updated in the tests as well.